packages/frontend/src/pages/Settings.tsx

```diff
@@ -189,124 +189,99 @@ const Settings: FC = () => {
+                                        exact
+                                        icon={
+                                            <MantineIcon
+                                                icon={IconBuildingSkyscraper}
+                                            />
+                                        }
+                                    />
+                                )}
 
-                                    {user.ability.can(
-                                        'update',
-                                        'OrganizationMemberProfile',
-                                    ) && (
-                                        <RouterNavLink
-                                            label={
-                                                isGroupManagementEnabled
-                                                    ? 'Users & groups'
-                                                    : 'User management'
-                                            }
-                                            to="/generalSettings/userManagement"
-                                            exact
-                                            icon={
-                                                <MantineIcon
-                                                    icon={IconUserPlus}
-                                                />
-                                            }
-                                        />
-                                    )}
-                                    {user.ability.can(
-                                        'manage',
-                                        subject('Organization', {
-                                            organizationUuid:
-                                                organization.organizationUuid,
-                                        }),
-                                    ) && (
-                                        <RouterNavLink
-                                            label={
-                                                isGroupManagementEnabled
-                                                    ? 'User & group attributes'
-                                                    : 'User attributes'
-                                            }
-                                            to="/generalSettings/userAttributes"
-                                            exact
-                                            icon={
-                                                <MantineIcon
-                                                    icon={IconUserShield}
-                                                />
-                                            }
-                                        />
-                                    )}
+                                {user.ability.can(
+                                    'update',
+                                    'OrganizationMemberProfile',
```

Comment:
These abilities are not checking organizationUuids, which is fine, for now, since we only allow 1 org per user

Reply:
same as above, not in the scope of the change. but I agree!